docs/core/tutorials/library-with-visual-studio.md

```diff
@@ -27,68 +36,83 @@ To create the blank solution:
 
-   1. On the **Configure your new project** page, enter **StringLibrary** in the **Project name** box. Then, choose **Create**.
+   1. Choose the **Console App (.NET Core)** template, and then choose **Next**.
 
-1. Check to make sure that the library targets the correct version of .NET Standard. Right-click on the library project in **Solution Explorer**, and then select **Properties**. The **Target Framework** text box shows that the project targets .NET Standard 2.0.
+   1. On the **Configure your new project** page, enter **ShowCase** in the **Project name** box. Then choose **Create**.
 
-   ![Project properties for the class library](./media/library-with-visual-studio/vb/library-project-properties.png)
+1. In **Solution Explorer**, right-click the **ShowCase** project and select **Set as StartUp Project** in the context menu.
 
-1. In the **Properties** dialog, clear the text in the **Root namespace** text box. For each project, Visual Basic automatically creates a namespace that corresponds to the project name. In this tutorial, you define a top-level namespace by using the [`namespace`](../../visual-basic/language-reference/statements/namespace-statement.md) keyword in the code file.
+   ![Visual Studio project context menu to set startup project](media/library-with-visual-studio/set-startup-project-context-menu.png)
 
-1. Replace the code in the code window with the following code and save the file:
+1. Initially, the new console app project doesn't have access to the class library. To allow it to call methods in the class library, create a project reference to the class library project. In **Solution Explorer**, right-click the `ShowCase` project's **Dependencies** node, and select **Add Project Reference**.
 
-   [!CODE-vb[ClassLib#1](../../../samples/snippets/core/tutorials/vb-library-with-visual-studio/stringlibrary.vb)]
+   ![Add reference context menu in Visual Studio](media/library-with-visual-studio/add-reference-context-menu.png)
 
-   The class library, `UtilityLibraries.StringLibrary`, contains a method named `StartsWithUpper`. This method returns a <xref:System.Boolean> value that indicates whether the current string instance begins with an uppercase character. The Unicode standard distinguishes uppercase characters from lowercase characters. The <xref:System.Char.IsUpper(System.Char)?displayProperty=nameWithType> method returns `true` if a character is uppercase.
+1. In the **Reference Manager** dialog, select the **StringLibrary** project, and select **OK**.
 
-1. On the menu bar, select **Build** > **Build Solution**.
+   ![Reference Manager dialog with StringLibrary selected](media/library-with-visual-studio/manage-project-references.png)
 
----
+1. In the code window for the *Program.cs* or *Program.vb* file, replace all of the code with the following code.
+
+   [!code-csharp[UsingClassLib#1](~/samples/snippets/csharp/getting_started/with_visual_studio_2017/showcase.cs)]
```

Comment:
Not necessary in this PR, but we should continue to migrate to the new snippet syntax when we update articles.